src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -220,7 +220,7 @@ private void testCreateInstructorAction() throws Exception {
         coursesPage.loadInstructorHomeTab();
         instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
         instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session")
-                          .waitForPageToLoad();
+                          .expandPanels();
```

Comment:
Ok, @boxin-yang, open another issue to skip the expanding for the other tests and make sure at least 1 test is testing for either expand / no expand (basically just cover both cases)